assets/app.js

```diff
@@ -13,3 +19,10 @@ $( '.close' ).on( 'click', function () {
 if ( window.localStorage.getItem( 'tutorial' ) === 'no' ) {
 	$( '.tutorial' ).hide();
 }
+
+$().ready( function () {
```

Comment:
Same as with the other PR, I think `$( function () {` will accomplish the same thing.

Reply:
Fixed.

I'm not quite sure this app.js file is as well-organised as it could be, but for now it's still small. Will refactor if required, later.